helper/src/network.rs

```diff
@@ -5,6 +5,11 @@
 //! into it's own crate.
 //!
 //! `#[no_std]` compatible.
+// TODO: move to types crate.
```

Comment:
Can you move this to `cuprate_types`? `strum` can automatically implement the `FromStr/Display` and lots of other stuff.

Also please add a doc-test asserting the string representations since we rely on it for PATHs now, something like this: https://github.com/Cuprate/cuprate/blob/63216aecaead915550be93236faf6aeeebf0e04f/rpc/types/src/misc/status.rs#L31-L65

Reply:
Will do in another PR this is used in a lot of places so don't really want to do it here.